pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+addopts = -vv --cov=fxa --flake8
```

Comment:
This could go to setup.cfg no?

Reply:
Yep this could indeed go to setup.cfg but some distributions later try to run tests in their packages and if I would put it to setup.cfg they would have to patch this stuff out or they would have to depend on coverage and flake. While here the sidst can either not contain pytest.ini or the packager can just remove the file as one of the buildsteps.